assets/js/components/adminbar/AdminBarWidgets.stories.js

```diff
@@ -140,6 +148,89 @@ Analytics4WidgetsLoading.args = {
 	},
 };
 
+export const ViewOnlyAnalyticsAndSearchConsole = Template.bind( {} );
+ViewOnlyAnalyticsAndSearchConsole.storyName =
+	'View Only Analytics And Search Console';
+ViewOnlyAnalyticsAndSearchConsole.args = {
+	setupRegistry: ( registry ) => {
+		provideModules( registry, [
+			{
+				slug: 'analytics',
+				active: true,
+				connected: true,
+			},
```

Comment:
Having `analytics` connected and active is not necessary any more, it can be removed.

Reply:
@zutigrm, we do need it to check if the Analytics module is connected. Otherwise, it will be disconnected.

 
<img width="1320" alt="Screenshot 2024-01-25 at 4 26 46 PM" src="https://github.com/google/site-kit-wp/assets/24408261/58f1ddce-7311-4e6c-a0b4-7049376f6252">

Reply:
Aha yes, my bad, `check-access` is still used against `analytics` module